src/FFMpeg/FFProbe.php

```diff
@@ -44,7 +44,7 @@ class FFProbe
     /** @var MapperInterface */
     private $mapper;
 
-    public function __construct(FFProbeDriver $ffprobe, Cache $cache)
+    public function __construct(FFProbeDriver $ffprobe, CacheItemPoolInterface $cache)
```

Comment:
This is BC Break, `Cache` does not implement `CacheItemPoolInterface`: https://github.com/doctrine/cache/blob/331b4d5dbaeab3827976273e9356b3b453c300ce/lib/Doctrine/Common/Cache/Cache.php